src/main/java/draughts/Board.java

```diff
@@ -49,32 +49,32 @@ public String toString() {
         Pawn [][] board = this.fields;
         StringBuilder result = new StringBuilder();
         Color background;
-        for(int i =0;i<board.length+1;i++){
-            for(int j=0;j<board[0].length+1;j++){
-                if(i>0&&j>0){
+        for(int rows = 0; rows < board.length + 1; rows++){ // i -> rows
```

Comment:
In my opinion it should be `row` instead of `rows`. The same comment for other for loops in this PR